Documentation/project-docs/triage.md

```diff
@@ -48,4 +48,25 @@ In particular, we strive to follow these guidelines:
 * **Do** close issues that represent work we don't think makes sense for the current or upcoming release
 * **Do** close issues that provide little or questionable value to APIs that are included with the .NET Framework
 * **Do** favor issues that do not require modifying types that are included with the .NET Framework.
-* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
\ No newline at end of file
+* **Avoid** franken design just to avoid adding APIs to types that are included with the .NET Framework
+
+## Porting to .NET Framework
+
+### Constraints for shipping in the .NET Framework
+ 
+Each release of .NET Framework ships to over one billion machines word-wide and installs as an *in-place* update, requiring each fix to meet a high level of quality and compatibility.  Each fix is reviewed extensively to determine which release if any is most appropriate.
+
+Here are some of the factors that are considered:
+
+* **Backward compatibility**. We do not want to break existing applications.
+* **Risk and/or size of the change**. We need to consider whether the change is on a common code path or causes a lot of churn.
+* **Measurable quality of the change**. We need to pay attention to our ability test the change in an complete end-to-end fashion.
+* **Value of the change**. All the risks outlined above need to be balanced against how many customers the change will help or how much the change will help customers.
+ 
+### How you can tell whether a change will be ported
+ 
+Generally, all changes are expected to be tracked as GitHub issues. We are going to introduce the following three labels on GitHub to track and report progress around porting:
+
+* **netfx-consider**: A given issue should be considered for inclusion in the .NET Framework. Once an issue has this label, the team will consider it for inclusion in the .NET Framework.
+* **netfx-approve**: If the proposed change is shown to add value and not introduce unmitigated risk then the issue will be checked in and ship in a future release of the .NET Framework. Marking an issue with this label does not imply that it will ship in the next release or any specific release of the .NET Framework.
+* **netfx-decline**: If the proposed change does not align with our business goals or introduces unmitigated risk, the issue will not be included in the .NET Framework.
```

Comment:
Agreed. I simply left this out because we haven't locked down the process how the reason will be communicated. It seems to say that we'll provide the reason without mentioning where.

Proposals?